Documentation/common-problems.md

```diff
@@ -234,26 +234,46 @@ What is happening here is that the MON pods are restarting and one or more of th
+# the mon pod is in a crash loop backoff state
+$ kubectl -n rook get pod
+NAME                   READY     STATUS             RESTARTS   AGE
+rook-ceph-mon0-r8tbl   0/1       CrashLoopBackOff   2          47s
+
+# the pod shows a termination status that the keyring does not match the existing keyring
+$ kubectl -n rook describe pod -l mon=rook-ceph-mon0
+...
+    Last State:		Terminated
+      Reason:		Error
+      Message:		The keyring does not match the existing keyring in /var/lib/rook/rook-ceph-mon0/data/keyring. 
+                    You may need to delete the contents of dataDirHostPath on the host from a previous deployment.
+...
+```
+
+If your cluster is larger than a couple nodes, you may get lucky enough that the monitors were able to start and form quorum. However, now the OSDs pods may fail to start due to state
+from a previous deployment. Looking at the OSD pod logs you will see an error about the file already existing.
+```
+$ kubectl -n rook logs rook-ceph-osd-fl8fs
+...
+2017-10-31 20:13:11.187106 I | mkfs-osd0: 2017-10-31 20:13:11.186992 7f0059d62e00 -1 bluestore(/var/lib/rook/osd0) _read_fsid unparsable uuid 
+2017-10-31 20:13:11.187208 I | mkfs-osd0: 2017-10-31 20:13:11.187026 7f0059d62e00 -1 bluestore(/var/lib/rook/osd0) _setup_block_symlink_or_file failed to create block symlink to /dev/disk/by-partuuid/651153ba-2dfc-4231-ba06-94759e5ba273: (17) File exists
+2017-10-31 20:13:11.187233 I | mkfs-osd0: 2017-10-31 20:13:11.187038 7f0059d62e00 -1 bluestore(/var/lib/rook/osd0) mkfs failed, (17) File exists
+2017-10-31 20:13:11.187254 I | mkfs-osd0: 2017-10-31 20:13:11.187042 7f0059d62e00 -1 OSD::mkfs: ObjectStore::mkfs failed with error (17) File exists
+2017-10-31 20:13:11.187275 I | mkfs-osd0: 2017-10-31 20:13:11.187121 7f0059d62e00 -1  ** ERROR: error creating empty object store in /var/lib/rook/osd0: (17) File exists
 ```
 
 ## Solution
-This is a common problem when reinitializing the Rook cluster and the local directory used for persistence has not been purged. This directory is the same directory reference by the `dataDiskHostPath` setting in the cluster CRD and is typically set to `/var/lib/rook`. To remedy simply shutdown all components of Rook and then delete the contents of `/var/lib/rook` or the directory specified by `dataDiskHostPath` on each of the hosts in the cluster. This time when the cluster CRD is applied, the rook-operator should start all the Pods as expected.
+This is a common problem reinitializing the Rook cluster when the local directory used for persistence has **not** been purged. This directory is the `dataDiskHostPath` setting in the cluster CRD and is typically set to `/var/lib/rook`. To fix the issue you will need to delete all components of Rook and then delete the contents of `/var/lib/rook` (or the directory specified by `dataDiskHostPath`) on each of the hosts in the cluster. Then when the cluster CRD is applied to start a new cluster, the rook-operator should start all the pods as expected.
```

Comment:
`dataDirHostPath` is the correct field name.  Please update here in all places.

Reply:
oh ha, i remember commenting on that in the original PR for this topic, but it fell through somehow